### DISCUSSION TITLE
authorizationParams in signIn/signout functions

### DISCUSSION THREAD

**Discussion by stychu** [nextauthjs/next-auth#5687] [Help]
Looking at signIn function I should be able to provide additional params but it seems it doesn't work or I miss something.

```js
 await signIn(undefined, undefined, "error=SessionRequired");
```

I want to pass `error` SessionRequired on given circumstances but doing like above doesn't seem to take any effect. Not mentioning singOut doesn;t event accept any additional url params which is lame

**Comment by fers4t** [comment]
Did you try it with object like
```js
await signIn(undefined, undefined, { userType: "company" }
```

It should be pass like this 
![image](https://user-images.githubusercontent.com/58539842/200133638-ddb5414c-f97d-4076-99f1-fb9a6614fe42.png)

**Reply by stychu** [reply]
Yes I tried. Doesn't work for me for some reason hmmm
